modules/audiencerunBidAdapter.js

```diff
@@ -0,0 +1,143 @@
+import * as utils from '../src/utils';
+import { config } from '../src/config';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+
+const BIDDER_CODE = 'audiencerun';
+const ENDPOINT_URL = 'https://d.audiencerun.com/prebid';
```

Comment:
Responses from this endpoint are not compressed, but are required to be. 

Adapter Requirements: http://prebid.org/dev-docs/bidder-adaptor.html#bidder-adaptor-Required-Adapter-Conventions

Reply:
we are already return content-encoding "gzip"